src/main/java/service/CheckUsersService.java

```diff
@@ -0,0 +1,23 @@
+package service;
+
+import models.User;
+
+import java.util.List;
+
+public class CheckUsersService {
+    private List<User>userList;
```

Comment:
we can make this field final to underline that at least the reference does not change after initialization.

Reply:
- [x] made List<User>userList final